schema/features/malicious-site-protection.ts

```diff
@@ -0,0 +1,11 @@
+import { Feature, CSSInjectFeatureSettings } from '../feature';
+
+type State = 'enabled' | 'disabled' | 'internal';
+type StateObject = {
+    state: State;
+};
```

Comment:
```suggestion
```

Reply:
(I think this is unused anyway, looks like we don't have a lint that prevents unused types)